templates/aws/template-variables.rc

```diff
@@ -0,0 +1,19 @@
+# Kubernetes cluster configuration
+export KUBERNETES_VERSION=v1.30.0
+export CONTROL_PLANE_MACHINE_COUNT=                 # e.g. 1
+export WORKER_MACHINE_COUNT=
+
+# AWS region
+export AWS_REGION=""                                # e.g. "us-east-2"
+
+# AWS machine configuration
+export AWS_CREATE_BASTION=                          # e.g. "true"
+export AWS_PUBLIC_IP=                               # e.g. "true"
+export AWS_CONTROL_PLANE_INSTANCE_TYPE=             # e.g. "t3.large"
+export AWS_NODE_INSTANCE_TYPE=                      # e.g. "t3.large"
+export AWS_CONTROL_PLANE_ROOT_VOLUME_SIZE=          # in Gi, e.g. 16
```

Comment:
nit: just starred at this `Gi` and scratched my head. 
Can you put `Gigabyte` here for slow folks like me? Thanks